### ISSUE TITLE
Feat : Discussion forum

### ISSUE THREAD

**Issue by sakshi-s** [Feminine-Divine/periodo#144]
Most of the time we shy away from talking about periods. The website could have a discussion forum where women can openly talk about their period experiences.

**Comment by khushishikhu** [comment]
@sakshi-s Good idea. can you please tell us briefly how are you planning to add this feature

**Comment by Aaishpra** [comment]
@sakshi-s are you going to work here,

**Comment by sakshi-s** [comment]
I have my exams, I can't work on it till the weekend.

**Comment by Aaishpra** [comment]
No problem take your time, just keep giving updates.
And all the best for your exams👍🏻

**Comment by raks924** [comment]
We can add  a discussion forum like Reddit app. And girls can have a chit chat.

**Comment by khushishikhu** [comment]
@sakshi-s if you are not interested , please tell us. So that I can assign this issue to @raks924

**Comment by khushishikhu** [comment]
@raks924 assigning this to you. Happy Coding!!

**Comment by A-HK** [comment]
Hey @khushishikhu @Aaishpra , is this issue still being worked on?
If not, would you mind assigning it to me?
